gf-guide/metadata.md

```diff
@@ -181,7 +181,7 @@ Typographic classification, one of 5 possible values:
 -   `category: "HANDWRITING"`
 -   `category: "MONOSPACE"`
 
-Since 2022, it is possible to assign a family with multiple categories (usually two). The second one assigned in the METADATA.pb would be the main one.
+This key is deprecated in the API (cf. below Stroke and Classification), but still mandatory to fill up in METADATA.pb
```

Comment:
Similar here, to my understanding, the key is not deprecated (yet) but is used to assign the main category of the font. Then stroke and classification are used to further define the font (e.g. adding a subcategory)

Reply:
Yes, the category field in intended to continue to exist with no changes to its semantics post-adding the stroke and classification fields.

We are keeping the category field because Google products like Workspace and Ads use them and we didn't want to affect those products.

Reply:
I pushed a new commit explaining why this is not deprecated.